sdk_extensions/zpages/src/test/java/io/opentelemetry/sdk/extensions/zpages/TraceConfigzZPageHandlerTest.java

```diff
@@ -154,7 +182,10 @@ void appliesChangesCorrectly_formSubmit() {
 
     TraceConfigzZPageHandler traceConfigzZPageHandler =
         new TraceConfigzZPageHandler(tracerProvider);
-    traceConfigzZPageHandler.emitHtml(queryMap, output);
+    boolean error = traceConfigzZPageHandler.processRequest("POST", queryMap, output);
+    if (!error) {
```

Comment:
Do we need to check this in all the tests? Aren't they non-error tests?

Reply:
Yea they're non-error tests, I replaced all with search, I'll removed the checks